.github/workflows/ci.yml

```diff
@@ -33,18 +33,16 @@ jobs:
       - name: Install dependencies
         shell: bash -l {0}
         run: |
-          conda install -c conda-forge pip botocore aiobotocore==2.1.0 "aiohttp<=4" "moto>=2.0,<3" pytest flake8 black -y
+          pip install -r requirements.txt -r test_requirements.txt
```

Comment:
It's a pretty big change to swap out conda (/mamba) installation for pip. My life usually revolves around conda, so I would like to keep that, but I would understand adding a separate run with pip.

Reply:
(an action stage can be made to use a conda environment file instead of listing dependencies right in the run file, which is probably what you were after)